install-from-github.js

```diff
@@ -27,6 +28,9 @@ try {
 
 console.log(`Downloading browsers...`);
 const { installBrowsersWithProgressBar } = require('./lib/install/installer');
-installBrowsersWithProgressBar(__dirname);
+installBrowsersWithProgressBar(__dirname).catch(e =>  {
+  console.error(`Failed to install browsers, caused by\n${e.stack}`);
+  exit(1);
```

Comment:
process.exit does it too. It's predefined and globally available. No need to import it afaik.

Reply:
done